tests/Integration/tests/test_add_articles.py

```diff
@@ -0,0 +1,115 @@
+"""
+This is an example automated test to help you learn Qxf2's framework
+Our automated test will do the following:
+    #Open Qxf2 newsletter generator application
+    #Fill the details of add articles section.
+
+"""
+import os,sys,time
+from turtle import title
+from typing_extensions import runtime
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from page_objects.PageFactory import PageFactory
+from utils.Option_Parser import Option_Parser
+import conf.mail_conf as conf
+import conf.add_articles_conf as conf
+import pytest
+
+@pytest.mark.GUI
+def test_add_article(test_obj):
+
+    "Run the test"
+    try:
+        #Initalize flags for tests summary
+        expected_pass = 0
+        actual_pass = -1
+        # Create a test object for add articles 
+        test_obj = PageFactory.get_page_object("add articles page")
+        #Set start_time with current time
+        start_time = int(time.time())
+        # test_obj.turn_on_highlight()
+        
+        #Get the test details from the conf file
+        email = conf.email
+        password = conf.password
+        
+        #Set the login
+        login = test_obj.login_page(email,password)
+        # click the hamburger button
+        hamburger_button = test_obj.click_hamburger_button()
+        # click the add_articles button
+        add_button = test_obj.click_addarticle()
+       
+        #3.Get the test details from the conf file and fill the forms
+        article_list = conf.article_list
+        article_number = 1		#Initalize form counter  
+        
+        # Collect form data
+        for article in article_list:
+            url = article['URL']
+            title = article['TITLE']
+            description = article['DESCRIPTION']
+            runtime = article['RUNTIME']
+            category = article['CATEGORY']
+            submit_button = test_obj.click_submit()
+            add_another_article = test_obj.click_addanother_article()
+           
+            msg ="\nReady to fill article number %d"%article_number
+            test_obj.write(msg)
+           
+            #  Visit main page again
+            test_obj = PageFactory.get_page_object("add articles page")
+            article_number = article_number + 1
```

Comment:
[Needs to be fixed]
This increment should happen only after you submit the article successfully, no?